hllplus/sparse.go

```diff
@@ -254,3 +309,18 @@ func (s *deltaSlice) Iterate(fn func(uint32)) {
 		last = x
 	})
 }
+
+func (s *deltaSlice) CopyBytes() []byte {
```

Comment:
we use `Clone` everywhere, but OK

Reply:
Well, Clone copies entire receiver; I wanted smth to get an immediate value of it's buffer.
Or you mean, let's do `deltaSlice.Clone().Bytes()`?

Reply:
I get that, I think the overhead `deltaSlice.Clone().Bytes()` is tiny and it's a) easier to read and b) less code needed